docs/docs/create-transformer-plugin.md

```diff
@@ -2,36 +2,272 @@
 title: Create a Transformer Plugin
 ---
 
-Transformer plugins are responsible for "transforming" data provided by
-[source plugins](/docs/create-source-plugin) into new nodes and/or node fields.
+There are two types of plugins that work within Gatsby's data system, "source"
+and "transformer" plugins.
 
-## Transformer plugin example
```

Comment:
Oh, forgot to mention that I wasn't clear on this header and section -- is this example provided because I'm supposed to imitate it? Knowing why this example is included would help clarify why it's introduced here

Reply:
Probably makes sense to remove it. We're defining what transformer plugins are before moving into the "create" content, but an example is probably not necessary. The leading section page intros this.